### PR TITLE
closed #9 #5 #10

### LINKED ISSUE #9: Update html structure of UpcomingMatches
Update component UpcomingMatches.js and its render function to produce following output:
upcomingmatches

Html structure can be copied from new.ksl.sk

AC:
- upcoming matches must have title with text "Najbližšie zápasy",
- upcoming matches must have text "Žiadne ďalšie zápasy nie sú naplánované."
- upcoming matches must have light gray background color
- there must be an border bottom under title
- put text in alert bootstrap component
- no custom css is allowed, use only existing bootstrap classes for styling

### PR DESCRIPTION
**Pull request by MJuhas**

### CODE REVIEW

File: src/components/UpcomingMatches.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import React, { PureComponent } from 'react';
 
+
 export default class UpcomingMatches extends PureComponent {
     render() {
-        return <div>Žiadne dalsie zápasy</div>;
+            return  <div class="bg-light">
```
Comment (buksy90) [inline src/components/UpcomingMatches.js]: Please fix the "class" attribute, make sure theres no errors in console

File: src/components/MainMenu.js [hunk 2]
```diff
@@ -64,13 +64,13 @@ export default class MainMenu extends PureComponent {
                     </button>
 
                     <div className={"collapse navbar-collapse" + (this.state.opened ? " show" : "")} id="navbarSupportedContent">
-                        <ul className="navbar-nav mr-auto">
-                            <li className="nav-item active"><Link to="/" className="nav-link">Úvod</Link></li>
-                            <li className="nav-item"><Link to="/schedule" className="nav-link">Rozpis</Link></li>
-                            <li className="nav-item"><a className="nav-link" href="/tabulka">Tabuľky</a></li>
-                            <li className="nav-item"><a className="nav-link" href="/playground">Ihriská</a></li>
+                        <ul className="navbar-nav w-100 d-flex ">
```
Comment (buksy90) [inline src/components/MainMenu.js]: + thanks for adding padding, it looks better
- why did you added d-flex? its not needed

File: src/components/MainMenu.js [hunk 3]
```diff
@@ -64,13 +64,13 @@ export default class MainMenu extends PureComponent {
...
+                            <li className="nav-item active p-2"><Link to="/" className="nav-link">Úvod</Link></li>
+                            <li className="nav-item p-2"><Link to="/schedule" className="nav-link">Rozpis</Link></li>
+                            <li className="nav-item p-2"><a className="nav-link" href="/tabulka">Tabuľky</a></li>
+                            <li className="nav-item p-2"><a className="nav-link" href="/playground">Ihriská</a></li>
                             <DropDown text="Liga" items={[{text: "Test1", link: "link1"}]}/>
-                            <li className="nav-item navbar-right"><a className="nav-link" href="/login/facebook">Prihlásiť</a></li>
+                            <li className="nav-item ml-auto p-2 "><a className="nav-link " href="/login/facebook">Prihlásiť</a></li>
```
Comment (buksy90) [inline src/components/MainMenu.js]: good solution with margin left

File: src/components/NewsList.js [hunk 4]
```diff
@@ -1,7 +1,17 @@
...
     render() {
-        return <div>Novinky</div>;
+        return <div>
+                    <div class="bg-light py-4"  >
+                        <div class=" text-white h5 bg-warning  mx-4   py-3 px-4">
+                                    Žiadne ďalšie zápasy nie sú naplánované
+                        </div>
+                    </div>
+                    <div class="my-3 mx-5 border-bottom" >
+                        <p class="display-2 font-weight-light">Novinky</p>
```
Comment (buksy90) [inline src/components/NewsList.js]: - This should be the main title of page, use H1

**Comment by buksy90** [comment]
I just merged another branch, there was also a change in MainMenu.js. Since you change that file too, there is a conflict. You need to rebase your branch above the master branch.
1. switch to master branch (git checkout master)
2. download latest updates (git pull)
3. switch back to your branch (git checkout newFeature)
4. check gitk --all and notice where your branch is, and where is master (master should be above)
5. rebase your branch (git rebase master), there might be some conflicts (but its possible there wont be any), in that case you need to open MainMenu.js file and resolve the conflicts, you need to update the file so that both your and mine code is kept and it works together. (after resolving conflict, run "git add ." and "git rebase --continue")
6. check gitk --all and notice your branch is above the master
7. upload your changes, you need to use force option because you're overwriting your previous commit (git push --force)

File: src/components/NewsList.js [hunk 5]
```diff
@@ -1,7 +1,17 @@
 import React, { PureComponent } from 'react';
 
+
 export default class NewsList extends PureComponent {
     render() {
-        return <div>Novinky</div>;
+        return <div>
+                    <div class="bg-light py-4"  >
+                        <div class=" text-white h5 bg-warning  mx-4   py-3 px-4">
+                                    Žiadne ďalšie zápasy nie sú naplánované
```
[the remark this answers is not in the document]
Comment (MJuhas) [reply src/components/NewsList.js]: I just learnig, i dont have enought experience and knowlage, and I dont know what i have got write suces code

**Pull request merged**